boards/saml10-xpro/doc.txt

```diff
@@ -51,14 +51,14 @@ memory.
 | Low-level driver | GPIO    | yes       | |
 |        | PWM       | no            | |
 |        | UART      | yes           | |
-|        | I2C       | no        | |
-|        | SPI       | no        | |
+|        | I2C       | yes        | |
+|        | SPI       | yes        | |
 |        | USB       | no        | |
-|        | RTT       | no        | |
-|        | RTC       | no       |  |
-|        | RNG       | no        |  |
+|        | RTT       | yes        | |
+|        | RTC       | yes      |  |
+|        | RNG       | yes        |  |
```

Comment:
This still requires #11647 to be merged, but then it would also be true for saml11-xpro.